### PR TITLE
fix: children of Tabs[type="editable-card"] cannot be falsy

### PR DESCRIPTION
**Pull request by oldturkey**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    if tabs type ="editable-card",if children has falsy value ,we will get error message,so we should fix it。 [demo](https://codesandbox.io/s/lively-glade-jmg5s ) here is error info      |
| 🇨🇳 Chinese |      当tabs组件使用type为"editable-card"时，如果子组件为falsy值会导致程序报错，因此增加了一个判断去修复这个bug     |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/tabs/demo/custom-add-trigger.md](https://github.com/oldturkey/ant-design/blob/master/components/tabs/demo/custom-add-trigger.md)

### CODE REVIEW

**Comment by oldturkey** [comment]
@zombieJ

File: components/tabs/demo/custom-add-trigger.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ class Demo extends React.Component {
               {pane.content}
             </TabPane>
           ))}
+          {undefined}
```
Comment (afc163) [inline components/tabs/demo/custom-add-trigger.md]: 不要改 demo，加一个用例吧。

Comment (oldturkey) [reply components/tabs/demo/custom-add-trigger.md]: 好的，我重新pr

Comment (afc163) [reply components/tabs/demo/custom-add-trigger.md]: 不需要重新 PR，继续在这里改就行了。

Comment (oldturkey) [reply components/tabs/demo/custom-add-trigger.md]: 原来会自动更新呀，刚发现。 去掉demo里加的东西啦

**Comment by afc163** [comment]
测试用例。

**Pull request merged**